conda/recipes/cuspatial/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - cudf {{ minor_version }}.*
     - rmm {{ minor_version }}.*
     - gdal >=3.2.0,<3.3.0a0
+    - geopandas >=0.9.0
```

Comment:
Should we pin this to `<0.10.0a0`? Have geopandas releases typically broken us?

Reply:
Yeah, actually, `geopandas` seems to be changing quickly with breaking changes each minor version.